src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -55,28 +77,101 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
   );
 
   return (
-    <StyledBylineSection role="region" aria-labelledby="article-byline">
-      <VisuallyHiddenText id="article-byline">
+    <section
+      css={BylineCss.bylineSection}
+      role="region"
+      aria-labelledby="article-byline"
+    >
+      <VisuallyHiddenText id="article-byline" aria-hidden>
         {articleInformationTranslated}
       </VisuallyHiddenText>
-      <BylineList role="list">
+      <ul css={BylineCss.bylineList} role="list">
         <li>
-          <span role="text">
-            <VisuallyHiddenText>{`${authorTranslated}, `} </VisuallyHiddenText>
-            <Author script={script} service={service}>
-              {author}
-            </Author>
-          </span>
+          {twitterLink ? (
+            <React.Fragment>
+              <VisuallyHiddenText>{`${authorTranslated}, `}</VisuallyHiddenText>
+              <a
+                css={[BylineCss.link, BylineCss.authorLink]}
+                href={twitterLink}
+                target="_blank"
```

Comment:
Please could we remove this for both links? It's not good for accessiblity to open links in new windows.